charts/add-ons/grafana/templates/grafana-rbac.yaml

```diff
@@ -11,3 +11,4 @@ metadata:
   labels:
     {{.Values.global.controllerComponentLabel}}: grafana
     {{.Values.global.controllerNamespaceLabel}}: {{.Values.global.namespace}}
+{{ include "partials.image-pull-secrets" . }}
```

Comment:
Please add use `{{- include "partials.image-pull-secrets" . }}` to suppress line-feeds/spaces (this is what's making the tests fail)